mailpoet/lib/Newsletter/AutomatedLatestContent.php

```diff
@@ -65,8 +65,13 @@ public function getPosts($args, $postsToExclude = [], $newsletterId = false, $ne
       'post_type' => (isset($args['contentType'])) ? $args['contentType'] : 'post',
       'post_status' => (isset($args['postStatus'])) ? $args['postStatus'] : 'publish',
       'orderby' => 'date',
-      'order' => ($args['sortBy'] === 'newest') ? 'DESC' : 'ASC',
+      'order' => isset($args['sortBy']) && in_array($args['sortBy'], ['newest', 'DESC']) ? 'DESC' : 'ASC',
```

Comment:
Was this broken? I don't know what the order params look like but is this a fix? It's probably something I'd normally extract to another commit to describe the fix in the commit message but of course no need to do it now.

Reply:
It was not broken per se, but it was wrong, it could easily force `DESC` to be interpreted as `ASC` and besides not checking if `sortBy` was set, I can put it to another commit